### ISSUE TITLE
Getting last_insert_id inside a transaction?

### ISSUE THREAD

**Issue by davechallis** [blackbeam/mysql_async#128]
Apologies if I've missed something, but I've been struggling to work out how to get the last insert ID of an autoincrement field inside a transaction.

I've got code like:

```
let pool = mysql_asyn::Pool::new("...my connection string...");
let mut conn = pool.get_conn().await?;
let mut tx = conn.start_transaction(TxOpts::default()).await?;

tx.exec_drop("INSERT INTO foo (value) VALUES (:value)", params! { "value": "some value" })
    .await?;

// how to get last insert ID here?
```

From what I can see, the `exec` methods return `Vec` of results rather than a `QueryResult`, so I can't access `last_insert_id()` that way, and I can't access it from the connection since it's been mutably borrowed by the transaction.

Any suggestions?

**Comment by blackbeam** [comment]
Hi. It's fixed (see #127), but not yet released. Sorry for that.
Please use `query_iter` + `last_insert_id` or `SELECT LAST_INSERT_ID`.

**Comment by davechallis** [comment]
Ah, great, I don't mind using the version from git for now anyway.
